app/interactors/comments/destroy.rb

```diff
@@ -0,0 +1,13 @@
+module Comments
+
+  class Destroy
+    include Interactor
+
+    delegate :comment, to: :context
+
+    def call
+      context.fail!(error: "Can't delete the comment!") unless comment.destroy
+      context.notice = "Comment has been deleted!"
```

Comment:
violates S from SOLID, you need to add this in controller, from locale for example